src/coreComponents/physicsSolvers/inducedSeismicity/DieterichSeismicityRate.cpp

```diff
@@ -0,0 +1,334 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file DieterichSeismicityRate.cpp
+ */
+
+// Source includes
+#include "DieterichSeismicityRate.hpp"
+
+namespace geos
+{
+
+namespace dataRepository
+{
+namespace keys
+{}
+}
+
+using namespace dataRepository;
+using namespace fields;
+
+/*----------------------------------------------------------------------------------
```

Comment:
I would edit out everything related to the documentation of the Laplace solver here, and explain what this solver does instead.

Reply:
Got it, I'll review what I might have left out in other places as well.